app/rsvp.tsx

```diff
@@ -38,11 +38,16 @@ export type RideDataType = {
   returns: Timestamp;
 };
 
+export type RideWithCreatorName = RideDataType & {
+  creatorName: string;
+};
+
```

Comment:
Since there's a finite number of accepted payment methods we should explicitly state what they are to avoid capitalization/spelling/hyphenation issues

```suggestion
  const AllowedPaymentMethods = "PayPal" | "Apple Cash" | "Venmo" | "Zelle" | "Google Pay" | "Cash App"
  type PaymentMethodsType = AllowedPaymentMethods[]
```